src/java.base/share/classes/java/io/ObjectInputFilter.java

```diff
@@ -546,6 +546,8 @@ static ObjectInputFilter rejectUndecidedClass(ObjectInputFilter filter) {
      * <p>If not `OVERRIDE`, the class must be public, must have a public zero-argument constructor, implement the
      * {@link BinaryOperator {@literal BinaryOperator<ObjectInputFilter>}} interface, provide its implementation and
      * be accessible via the {@linkplain ClassLoader#getSystemClassLoader() application class loader}.
+     * If the filter factory constructor is not invoked successfully, an {@link ExceptionInInitializerError}
+     * is thrown.
```

Comment:
Should you also say that later attempts to create an `ObjectInputStream` or to call `ObjectInputStream::setObjectInputFilter` will result in an `IllegalStateException`?